service/static/css/service.css

```diff
@@ -87,12 +87,12 @@
 
 .tab-nav li:hover {
     background-color: #ffffff !important;
-    color: #5fb8ba;
+    color: #8ad6e0;
```

Comment:
Hover colour wasn't "up for grabs" in the card. Need to verify the hover colours of tabs and links.